### DISCUSSION TITLE
In the CP, filter drop-down controls should show as much of the filter value as possible

### DISCUSSION THREAD

**Discussion by PaulBaileyAtChan** [ExpressionEngine/ExpressionEngine#4581] [Feature Requests]
### Suggest an idea for ExpressionEngine

A small UX/UI suggestion. In the CP — notably the Entry Manager and File Manager — when a filter value is too long to be displayed in the drop-down control, an ellipsis is displayed instead. In the screenshot, both the Channel and Status filters have values, but neither is actually displayed. This makes it much harder to see what's being filtered.

Suggestion is to display as much of the filter value as possible, truncating where necessary. In some/most cases, this will be enough to see what the filter value is.

<img width="720" alt="Screenshot 2024-11-08 at 10 26 17 AM" src="https://github.com/user-attachments/assets/420edf8f-a657-4ffe-8c86-ac826ca817f2">

**Comment by intoeetive** [comment]
Confirming this to be true on relatively small screens (or when the filter value is long enough)

What's worse is that when you open the dropdown, the selected value is not highlighted, so it's impossible to tell which value is selected for the filter. Maybe at the least we can do that (apply some checkmark for selected value in the dropdown)

@Yulyaswan

**Comment by Yulyaswan** [comment]
PR #4635

**Comment by TomJaeger** [comment]
This is shipping in 7.5.8
